EXECUTOR_LOGOPTIONS - TTCN Logger v2.2 options: TimeStampFormat:=Time; LogEntityName:=No; LogEventTypes:=Subcategories; SourceInfoFormat:=Stack; *.FileMask:=TESTCASE; *.ConsoleMask:=TESTCASE; LogFileSize:=0; LogFileNumber:=1; DiskFullAction:=Error
EXECUTOR_RUNTIME - Connected to MC.
EXECUTOR_RUNTIME - Executing test case tc_user in module EmergencyLoggingTest.
TESTCASE_START EmergencyLoggingTest.ttcn:34 Test case tc_user started.
PORTEVENT_STATE EmergencyLoggingTest.ttcn:34 Port internal_port was started.
PORTEVENT_STATE EmergencyLoggingTest.ttcn:34 Port external_port was started.
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:35 Before the loop
ACTION_UNQUALIFIED EmergencyLoggingTest.ttcn:36 Action: >>>>ACTION in tc_user, before the loop<<<<<
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:38 tc_user loop 0
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:38 tc_user loop 1
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:38 tc_user loop 2
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:38 tc_user loop 3
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:38 tc_user loop 4
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:38 tc_user loop 5
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:38 tc_user loop 6
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:38 tc_user loop 7
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:38 tc_user loop 8
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:38 tc_user loop 9
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:38 tc_user loop 10
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:38 tc_user loop 11
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:38 tc_user loop 12
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:38 tc_user loop 13
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:38 tc_user loop 14
ACTION_UNQUALIFIED EmergencyLoggingTest.ttcn:40 Action: >>>>ACTION in tc_user, after the loop<<<<<
VERDICTOP_SETVERDICT EmergencyLoggingTest.ttcn:41 setverdict(pass): none -> pass
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:42 This line reached last
PORTEVENT_STATE EmergencyLoggingTest.ttcn:42 Port internal_port was stopped.
PORTEVENT_STATE EmergencyLoggingTest.ttcn:42 Port external_port was stopped.
EXECUTOR_RUNTIME EmergencyLoggingTest.ttcn:42 Waiting for PTCs to finish.
VERDICTOP_FINAL EmergencyLoggingTest.ttcn:42 Setting final verdict of the test case.
VERDICTOP_FINAL EmergencyLoggingTest.ttcn:42 Local verdict of MTC: pass
VERDICTOP_FINAL EmergencyLoggingTest.ttcn:42 No PTCs were created.
TESTCASE_FINISH EmergencyLoggingTest.ttcn:42 Test case tc_user finished. Verdict: pass
EXECUTOR_RUNTIME - Executing test case tc_PTC_create_EL in module EmergencyLoggingTest.
TESTCASE_START EmergencyLoggingTest.ttcn:57 Test case tc_PTC_create_EL started.
PORTEVENT_STATE EmergencyLoggingTest.ttcn:57 Port internal_port was started.
PORTEVENT_STATE EmergencyLoggingTest.ttcn:57 Port external_port was started.
PARALLEL_UNQUALIFIED EmergencyLoggingTest.ttcn:59 Creating new PTC with component type Titan_LogTestDefinitions.MTCType.
PARALLEL_UNQUALIFIED EmergencyLoggingTest.ttcn:60 Creating new PTC with component type Titan_LogTestDefinitions.MTCType.
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:62 Last line before the DTE
ERROR_UNQUALIFIED EmergencyLoggingTest.ttcn:63 Dynamic test case error: Assignment of an unbound integer value.
VERDICTOP_SETVERDICT EmergencyLoggingTest.ttcn:63 setverdict(error): none -> error
EXECUTOR_RUNTIME EmergencyLoggingTest.ttcn:63 Performing error recovery.
PORTEVENT_STATE EmergencyLoggingTest.ttcn:63 Port internal_port was stopped.
PORTEVENT_STATE EmergencyLoggingTest.ttcn:63 Port external_port was stopped.
EXECUTOR_RUNTIME EmergencyLoggingTest.ttcn:63 Waiting for PTCs to finish.
VERDICTOP_FINAL EmergencyLoggingTest.ttcn:63 Setting final verdict of the test case.
VERDICTOP_FINAL EmergencyLoggingTest.ttcn:63 Local verdict of MTC: error
VERDICTOP_FINAL EmergencyLoggingTest.ttcn:63 Local verdict of PTC with component reference 3: none (error -> error)
VERDICTOP_FINAL EmergencyLoggingTest.ttcn:63 Local verdict of PTC with component reference 4: none (error -> error)
TESTCASE_FINISH EmergencyLoggingTest.ttcn:63 Test case tc_PTC_create_EL finished. Verdict: error
EXECUTOR_RUNTIME - Executing test case tc_user_EL in module EmergencyLoggingTest.
TESTCASE_START EmergencyLoggingTest.ttcn:17 Test case tc_user_EL started.
PORTEVENT_STATE EmergencyLoggingTest.ttcn:17 Port internal_port was started.
PORTEVENT_STATE EmergencyLoggingTest.ttcn:17 Port external_port was started.
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:21 Before the loop
ACTION_UNQUALIFIED EmergencyLoggingTest.ttcn:22 Action: >>>>ACTION in tc_user_EL, before the loop<<<<<
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 0
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 1
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 2
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 3
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 4
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 5
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 6
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 7
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 8
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 9
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 10
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 11
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 12
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 13
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 14
ACTION_UNQUALIFIED EmergencyLoggingTest.ttcn:26 Action: >>>>ACTION in tc_user_EL, after the loop<<<<<
VERDICTOP_SETVERDICT EmergencyLoggingTest.ttcn:27 setverdict(pass): none -> pass
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:28 This line is reached last
ERROR_UNQUALIFIED EmergencyLoggingTest.ttcn:29 Dynamic test case error: Assignment of an unbound integer value.
VERDICTOP_SETVERDICT EmergencyLoggingTest.ttcn:29 setverdict(error): pass -> error
EXECUTOR_RUNTIME EmergencyLoggingTest.ttcn:29 Performing error recovery.
PORTEVENT_STATE EmergencyLoggingTest.ttcn:29 Port internal_port was stopped.
PORTEVENT_STATE EmergencyLoggingTest.ttcn:29 Port external_port was stopped.
EXECUTOR_RUNTIME EmergencyLoggingTest.ttcn:29 Waiting for PTCs to finish.
VERDICTOP_FINAL EmergencyLoggingTest.ttcn:29 Setting final verdict of the test case.
VERDICTOP_FINAL EmergencyLoggingTest.ttcn:29 Local verdict of MTC: error
VERDICTOP_FINAL EmergencyLoggingTest.ttcn:29 No PTCs were created.
TESTCASE_FINISH EmergencyLoggingTest.ttcn:29 Test case tc_user_EL finished. Verdict: error
EXECUTOR_RUNTIME - Executing test case tc_user_EL in module EmergencyLoggingTest.
TESTCASE_START EmergencyLoggingTest.ttcn:17 Test case tc_user_EL started.
PORTEVENT_STATE EmergencyLoggingTest.ttcn:17 Port internal_port was started.
PORTEVENT_STATE EmergencyLoggingTest.ttcn:17 Port external_port was started.
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:21 Before the loop
ACTION_UNQUALIFIED EmergencyLoggingTest.ttcn:22 Action: >>>>ACTION in tc_user_EL, before the loop<<<<<
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 0
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 1
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 2
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 3
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 4
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 5
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 6
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 7
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 8
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 9
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 10
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 11
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 12
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 13
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 14
ACTION_UNQUALIFIED EmergencyLoggingTest.ttcn:26 Action: >>>>ACTION in tc_user_EL, after the loop<<<<<
VERDICTOP_SETVERDICT EmergencyLoggingTest.ttcn:27 setverdict(pass): none -> pass
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:28 This line is reached last
ERROR_UNQUALIFIED EmergencyLoggingTest.ttcn:29 Dynamic test case error: Assignment of an unbound integer value.
VERDICTOP_SETVERDICT EmergencyLoggingTest.ttcn:29 setverdict(error): pass -> error
EXECUTOR_RUNTIME EmergencyLoggingTest.ttcn:29 Performing error recovery.
PORTEVENT_STATE EmergencyLoggingTest.ttcn:29 Port internal_port was stopped.
PORTEVENT_STATE EmergencyLoggingTest.ttcn:29 Port external_port was stopped.
EXECUTOR_RUNTIME EmergencyLoggingTest.ttcn:29 Waiting for PTCs to finish.
VERDICTOP_FINAL EmergencyLoggingTest.ttcn:29 Setting final verdict of the test case.
VERDICTOP_FINAL EmergencyLoggingTest.ttcn:29 Local verdict of MTC: error
VERDICTOP_FINAL EmergencyLoggingTest.ttcn:29 No PTCs were created.
TESTCASE_FINISH EmergencyLoggingTest.ttcn:29 Test case tc_user_EL finished. Verdict: error
EXECUTOR_RUNTIME - Executing test case tc_user_EL in module EmergencyLoggingTest.
TESTCASE_START EmergencyLoggingTest.ttcn:17 Test case tc_user_EL started.
PORTEVENT_STATE EmergencyLoggingTest.ttcn:17 Port internal_port was started.
PORTEVENT_STATE EmergencyLoggingTest.ttcn:17 Port external_port was started.
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:21 Before the loop
ACTION_UNQUALIFIED EmergencyLoggingTest.ttcn:22 Action: >>>>ACTION in tc_user_EL, before the loop<<<<<
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 0
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 1
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 2
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 3
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 4
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 5
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 6
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 7
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 8
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 9
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 10
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 11
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 12
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 13
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:24 tc_user_EL loop 14
ACTION_UNQUALIFIED EmergencyLoggingTest.ttcn:26 Action: >>>>ACTION in tc_user_EL, after the loop<<<<<
VERDICTOP_SETVERDICT EmergencyLoggingTest.ttcn:27 setverdict(pass): none -> pass
USER_UNQUALIFIED EmergencyLoggingTest.ttcn:28 This line is reached last
ERROR_UNQUALIFIED EmergencyLoggingTest.ttcn:29 Dynamic test case error: Assignment of an unbound integer value.
TESTCASE_FINISH EmergencyLoggingTest.ttcn:29 Test case tc_user_EL finished. Verdict: error
